source/index.ts

```diff
@@ -81,12 +82,13 @@ async function ipLocation(ip: string): Promise<ipLocation.LocationData> {
 		throw new TypeError("A valid ipv4 address must be provided!")
 	}
 
-	const { latitude, longitude, city, region, region_code, country_name, country_code, country_code_iso3, country_capital, country_tld, country_population, country_calling_code, continent_code, in_eu, postal, timezone, utc_offset, currency, currency_name, languages, country_area }: IpApiData = await ky(`https://ipapi.co/${ip}/json/`).json()
+	const { latitude, longitude, city, reserved, region, region_code, country_name, country_code, country_code_iso3, country_capital, country_tld, country_population, country_calling_code, continent_code, in_eu, postal, timezone, utc_offset, currency, currency_name, languages, country_area }: IpApiData = await ky(`https://ipapi.co/${ip}/json/`).json()
 
 	return {
 		latitude,
 		longitude,
 		city,
+		reserved: !!reserved,
```

Comment:
If `reserved` is true, we should return an object that only includes it instead of also returning lots of undefined values.